app/services/support_interface/who_ran_which_export_export.rb

```diff
@@ -0,0 +1,18 @@
+module SupportInterface
+  class WhoRanWhichExportExport
+    def data_for_export
+      DataExport
+      .where
+      .not(export_type: nil)
```

Comment:
Do we still have any?

Reply:
Yes, for ones that have been deprecated.